packages/shared/src/utils/date-helpers.tsx

```diff
@@ -0,0 +1,6 @@
+const convertDateToMonthTimestamp = (dateInput: string | Date): number => {
+  const date = new Date(dateInput);
+  return Date.UTC(date.getUTCFullYear(), date.getUTCMonth(), 1) / 1000;
+};
+
+export { convertDateToMonthTimestamp };
```

Comment:
The idea of this package is to share function between different packages - it's something I wanted to do LONG time ago - because I have duplicated code between functions and web ... however I just found out that create-react-scripts library doesn't support including typescript code from another packages in yarn ... fuckin BULLSHIT

But we could use this to share code between the functions package and app package. We could share the typescript defintions. For example this file packages/functions/src/libs/types.ts has lot of types for the API objects.

Reply:
Yes, I thought about that too. When I created the app I decided to not touch any other package until it is accepted, but now code sharing could also be considered between web and app for faction and country flag aswell